pom.xml

```diff
@@ -483,6 +487,13 @@
 
   <profiles>
     <profile>
+      <id>scala-2.11</id>
+      <properties>
+        <scala.version>2.11.2</scala.version>
```

Comment:
We are up to 2.11.5 now and `brew` has 2.11.4. We might want to bump this up before we commit it?